### PR TITLE
Changing analysis poll check to request 5 analysis at the same time

### PR DESCRIPTION
**Pull request by xavijam**
cc @javierarce @saleiva.

### CODE REVIEW

**Comment by saleiva** [comment]
\0/

File: lib/assets/javascripts/cartodb/common/background_polling/models/analysis_collection.js [hunk 1]
```diff
@@ -29,20 +30,30 @@ module.exports = Backbone.Collection.extend({
   },
 
   pollCheck: function() {
-    if (this._nextAnalysis) {
-      this._nextAnalysis.unbind(null, null, this);
+    if (this._nextAnalysisItems) {
+      _.each(this._nextAnalysisItems, function(mdl) {
+        mdl.unbind(null, null, this);
+      }, this);
     }
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/common/background_polling/models/analysis_collection.js]: Basically it gets the first X items which are not analyzed yet, and then we request the analysis. When all of them are analyzed, we get the next X items.

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5033/))

**Comment by javierarce** [comment]
:cool: :sunglasses:

**Pull request merged**